.ci/ci.yml.tmpl

```diff
@@ -284,12 +323,26 @@ jobs:
               # Every time a change runs through this pipeline, it will generate a commit with
               # a different hash - the hash includes timestamps.  Therefore, even if there's no
               # code diff, this push will update terraform's pending PR on every update to the
-              # magic-modules PR.  With this 'only_if_diff' feature, if the change to the 
+              # magic-modules PR.  With this 'only_if_diff' feature, if the change to the
               # magic-modules PR does not require an update to the terraform PR, this will
               # not push the update even though the commit hashes are different.
               only_if_diff: true
               force: true
         {% endif %}
+        {% if ansible_enabled %}
+          - put: ansible-intermediate
+            params:
+              repository: magic-modules-with-comment/build/ansible
+              branch_file: magic-modules-with-comment/original_pr_branch_name
+              # Every time a change runs through this pipeline, it will generate a commit with
```

Comment:
Can skip the comment the second time - easy for them to get out of sync.  "See comment on terraform push" will do.